yarn.lock

```diff
@@ -0,0 +1,7364 @@
+# THIS IS AN AUTOGENERATED FILE. DO NOT EDIT THIS FILE DIRECTLY.
```

Comment:
I'm all for using yarn but if we add this, let's delete the package-lock.json file.

Reply:
This was added by mistake, I've removed